packages/frontend-shared/src/components/Tooltip.vue

```diff
@@ -2,7 +2,11 @@
   <Tooltip
     v-if="!disabled"
     :popper-class="{'no-arrow': hideArrow}"
-    :popper-hide-triggers="['hover']"
+    :popper-triggers="['hover']"
+    :hide-triggers="['hover']"
+    :auto-hide="!isInteractive"
+    :theme="theme"
+    :placement="placement ?? 'auto'"
```

Comment:
I did a little more testing and found what I think are some other side effects related to these changes that surface when using the keyboard, leading to unexpected states:

![Screen Shot 2022-05-24 at 6 39 19 PM](https://user-images.githubusercontent.com/8340719/170148942-732a9fe4-d76a-4241-8233-397d536a4b36.png)

After comparing tooltips in general with 10.0-release, it seems something in here is also affecting the tooltips in the Selector Playground - both their positioning and focus behavior. Moving focus out to the address bar immediately when a button that has a tooltip is focused. Let me know if you can reproduce, here's what I did:

https://www.loom.com/share/598305649aa741499ca1f909b80e31db - 10.0-release
https://www.loom.com/share/469f94e459554d729c5fe13b63489f2a - this branch

This seem like pretty extreme effects for such a small set of changes, so maybe something else is up with my local env. Though the changed positioning in the Selector Playground is also captured in percy [here](https://percy.io/cypress-io/cypress/builds/18347831/changed/1028975721?browser=chrome&browser_ids=23&subcategories=unreviewed%2Cchanges_requested&viewLayout=overlay&viewMode=new&width=1000&widths=536%2C800%2C1000%2C1500).